src/components/checkbox/checkbox-validations.stories.tsx

```diff
@@ -0,0 +1,311 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+import { Checkbox, CheckboxGroup } from ".";
+
+export const StringValidation: ComponentStory<typeof Checkbox> = () => (
+  <>
+    <Checkbox
+      error="Message"
+      id="checkbox_error"
+      key="checkbox_error"
+      label="Example checkbox (error)"
+      name="checkbox_error"
+    />
+    <Checkbox
+      id="checkbox_warning"
+      key="checkbox_warning"
+      label="Example checkbox (warning)"
+      name="checkbox_warning"
+      warning="Message"
+    />
+    <Checkbox
+      id="checkbox_info"
+      info="Message"
+      key="checkbox_info"
+      label="Example checkbox (info)"
+      name="checkbox_info"
+    />
+  </>
+);
+
+export const StringValidationWithTooltipPosition: ComponentStory<
```

Comment:
Instead of having multiple stories that are nearly identical, you could always define 1 template story and then generate new ones based on that template. 

Take a look at `DefaultStory` and for example `Disabled`, `Empty` and `Required` stories to see how it can be done
https://github.com/Sage/carbon/pull/5517/files#diff-145ae5850fb206056a2ce09868044bd4f1bd5e96a4b55b35982d95ddef9efe95R7

Reply:
unfortunately we cannot do that in this case, as every checkbox has different argument (error, warning, info).